### PR TITLE
Bitrise step install bundler 1.1.1

### PR DESCRIPTION
**Pull request by igorcferreira**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by igorcferreira** [comment]
Hello, could any one please inform me why the CI failed?

**Comment by trapacska** [comment]
Hi, the error is herE:

```
- (3/4) Running: Ensure all type_tags and project_type_tags are supported
 > Failed, error:
   - invalid type_tag: xcode
```

File: steps/bitrise-step-install-bundler/1.1.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  git: https://github.com/FutureWorkshops/bitrise-step-install-bundler.git
+  commit: 262d2d63ade6cbd03861e006f53c7bb5e45b8573
+host_os_tags:
+- osx-10.10
+- osx-10.9
+project_type_tags:
+- ios
+- xamarin
+type_tags:
+- xcode
```
Comment (trapacska) [inline steps/bitrise-step-install-bundler/1.1.1/step.yml]: This is an invalid type_tag. Please read our step development guideline.

**Comment by trapacska** [comment]
Please do a fresh share with the fix(es). I am closing this PR until. 

Thank you for the PR! 🚀

**Comment by igorcferreira** [comment]
Thanks for the info @trapacska and sorry for my mistake. I’ll fix it and re-open

**Pull request closed without merging**